### DISCUSSION TITLE
Two different endpoints returning the same response

### DISCUSSION THREAD

**Discussion by RajuJosephK** [mockoon/mockoon#443] [Help] [answered]
I have set the below mock end points 

PATCH  : resourceful/city/* /town/ *
Response ("success": "City-Town")

PATCH  :resourceful/city/* /town/*/area/ *
Response ("success": "City-Town-Area")

when I make a call with  https://localhost:3001/resourceful/city/123/town/10
I get the response ("success": "City-Town")  which is correct

when I make a call with  https://localhost:3001/resourceful/city/123/town/10/area/20
I get the response ("success": "City-Town")  instead of ("success": "City-Town-Area")

Can you help how I can achieve this

**Comment by 255kb** [comment, marked as answer]
The `*` is acting like the equivalent regex `.*`. This is not taking in account the `/` you may have, which means `.../town/*` will act like a catch-all. 
What you want to use here are route parameters: 
`resourceful/city/:city/town/:town/area/:area`. Then you can access these params in the rules, or with the `urlParam` templating helper: `{{urlParam 'route'}}`.

If you want more information about how the routing works, you can check [ExpressJS documentation](https://expressjs.com/en/guide/routing.html).
